internal/configload/configload.go

```diff
@@ -36,20 +36,23 @@ type config struct {
 	Dir     string        `yaml:"dir"`
 	Args    []string      `yaml:"args"`
 	Results struct {
-		Common   *testConfig `yaml:"common"`
-		FerretDB *testConfig `yaml:"ferretdb"`
-		MongoDB  *testConfig `yaml:"mongodb"`
+		Includes map[string][]string `yaml:"includes"`
+		Common   *testConfig         `yaml:"common"`
+		FerretDB *testConfig         `yaml:"ferretdb"`
+		SQLite   *testConfig         `yaml:"sqlite"`
+		MongoDB  *testConfig         `yaml:"mongodb"`
 	} `yaml:"results"`
 }
 
 // testConfig represents the YAML-based configuration for database-specific test configurations.
 type testConfig struct {
-	Default *ic.Status `yaml:"default"`
-	Stats   *stats     `yaml:"stats"`
-	Fail    []any      `yaml:"fail"`
-	Skip    []any      `yaml:"skip"`
-	Pass    []any      `yaml:"pass"`
-	Ignore  []any      `yaml:"ignore"`
+	Default     *ic.Status `yaml:"default"`
+	Stats       *stats     `yaml:"stats"`
+	Fail        []any      `yaml:"fail"`
+	Skip        []any      `yaml:"skip"`
+	Pass        []any      `yaml:"pass"`
+	Ignore      []any      `yaml:"ignore"`
+	IncludeFail []any      `yaml:"include_fail"`
```

Comment:
I wonder if all those can and should be `[]string`

Reply:
We cannot do that unfortunately as we use nested mappings for regex patterns.

Reply:
Unless I just use the same approach and unmarshal them into a field called `IncludeRegexp`. I will see.

Reply:
I added an `IncludeSkip` field to simplify how we handle regex patterns. But [diff.yml](https://github.com/FerretDB/dance/blob/main/tests/diff.yml) now breaks as we do not use type assertion so we cannot have nested mappings.

I propose we attempt to change the diff config in the near future instead. @AlekSi what do you think?